packages/app-js/src/index.tsx

```diff
@@ -55,22 +73,34 @@ class App extends React.PureComponent<Props, State> {
             className='js--Dropdown'
             onChange={(value) => this.selectExample(value)}
             options={snippets.map(({ code, ...options }) => ({ ...options }))}
-            label={'Select example'}
+            label={t('Select example')}
             defaultValue={snippet}
             withLabel
           />
       </header>
         <section className='js--Content'>
-          <Editor className='js--Editor' code={code} snippet={snippet} onEdit={this.onEdit}>
-            <Button
-              className='action-button'
-              isCircular
-              isPrimary
-              icon='play'
-              onClick={this.runJs}
-            />
+          <Editor
+            code={code}
+            snippet={snippet}
+            onEdit={this.onEdit}
+          >
+            <div className='action-button'>
+              <Button
+                isCircular
+                isPrimary
+                icon='play'
+                onClick={this.runJs}
+              />
+              <Button
+                isCircular
+                isDisabled={!isRunning}
+                isNegative
+                icon='close'
+                onClick={this.stopJs}
+              />
+            </div>
           </Editor>
-          <Output className='js--Output' logs={logs}>
+          <Output logs={logs}>
```

Comment:
`className='js--Output'` got lost here. Some of the styling is relying on this class.

Reply:
Ahh, it is just removing duplication - it is applied in the `Output` (and `Editor`) component already, so the above was actually a duplication.

Reply:
Ah, ok, closing!